layouts/partials/components/support-docs.html

```diff
@@ -2,20 +2,54 @@
     {{- partial "components/search-modal" . -}}
     <div class="container">
         <div class="row top-section">
-
             <div class="col-sm-12">
                 <h2 class="section-heading front-page-heading">Rackspace Support Documentation</h2>
                 <p class="front-blurb">The Rackspace Support Documentation provides guidance for users of all
                     Rackspace services.</p>
-                <div class="campaign-link" id="general-campaign">
+                <!-- <div class="campaign-link" id="general-campaign">
```

Comment:
This entire file is about the initial landing page layout. I assume you don't want these changes within this PR, correct? I guess you accidentally saved these changes on to this User Management branch.